controlpanel/frontend/jinja2/webapp-detail.html

```diff
@@ -176,9 +172,9 @@ <h2 class="govuk-heading-m">
     </tfoot>
   </table>
 
-{% if request.user.is_superuser %}
+{% if request.user.has_perm('api.add_app_admin', object) %}
   {% if admin_options|length %}
-  <form action="{{ url('add-app-admin', kwargs={ "pk": app.id }) }}" method="post" class="govuk-form-group">
+  <form action="{{ url('add-app-admin', kwargs={ "pk": object.id }) }}" method="post" class="govuk-form-group">
```

Comment:
is object still an `app`? `object` is quite generic and may easily lead to bugs next time we tweak this view.

Reply:
`object` is the standard context variable set by the generic view class, but you are correct, [the documentation recommends changing this](https://docs.djangoproject.com/en/2.2/topics/class-based-views/generic-display/#making-friendly-template-contexts) - i'll fix it